### PR TITLE
New Feature: Adds a page that shows the Full Inventory

### PR DESCRIPTION
**Pull request by SpeedyD**
It's purely in Home, as it's for a user to see, in one easier glance, exactly which items they have where.

It's similar to Account Search, but instead of making you look through items you may not even have, it simply shows everything you do have.

### CODE REVIEW

**Review by itinerare — APPROVED** [review]

File: app/Http/Controllers/Users/InventoryController.php [hunk 1]
```diff
@@ -198,6 +198,62 @@ public function getAccountSearch(Request $request) {
...
+    /**
+     * Show the full inventory page.
+     *
+     * @return \Illuminate\Contracts\Support\Renderable
+     */
+    public function getFullInventory() {
+        $user = Auth::user();
+
+        // Gather the user's characters and the items they own
+        $characters = Character::where('user_id', $user->id)->orderBy('slug', 'ASC')->get();
```
Comment (AW0005) [inline app/Http/Controllers/Users/InventoryController.php:210]: Why not use $user->characters or $user->allCharacters? And should this filter out hidden characters?

**Review by AW0005 — APPROVED** [review]
Had one comment but overall looks good!

Also I kind of like this inventory view style better. :0
Wonder if it might be nice to factor it out as something that could be toggled on for the regular inventory?

File: app/Http/Controllers/Users/InventoryController.php (hunk 1 above)
Comment (itinerare) [reply app/Http/Controllers/Users/InventoryController.php:210]: Good point-- I *do* think it should filter out hidden chars.

Comment (SpeedyD) [reply app/Http/Controllers/Users/InventoryController.php:210]: I based this off of the (Home) Inventory, Character Inventory and Account Search.. that is why.

I'll look into $user->allCharacters, that might be a better call.

**Review by AW0005 — APPROVED** [review]

**Pull request merged**